gpii/node_modules/matchMakerFramework/src/MatchMakerFramework.js

```diff
@@ -81,6 +80,158 @@
             event.fire(matchData, matchMakerPayload);
         });
     };
+	
+    /*
+	 * Returns a string indicating the matchmaker to select from configuration. Depends on the matchmaker payload.
+     */
+    gpii.matchMakerFramework.selectMatchMaker = function (matchMakerPayload) {
+		var result = gpii.matchMakerFramework.GetPreferredMatchMaker(matchMakerPayload);
+		return (result) ? result : gpii.matchMakerFramework.GetCapableMatchMaker(matchMakerPayload);
```

Comment:
This can also be written as `return result || gpii.matchMakerFramework.GetCapableMatchMaker(matchMakerPayload)`
(but just an FYI... either way is fine)